tests/run-tests

```diff
@@ -233,6 +234,11 @@ def run_tests(pyb, tests, args, base_path="."):
     if native == b'CRASH':
         skip_const = True
 
+    # Check if __rOP__ special methods are supported, and skip such tests if it's not
+    native = run_feature_check(pyb, args, base_path, 'reverse_ops.py')
```

Comment:
The use of "native" here is a cut and paste error from long ago: the first feature check here is for native support, so that's why the variable is called "native", and then it was just copied to the other feature checks.  It should be called either simply "output" or something related to the test, like "reverse".

Reply:
Yeah, and overall, run-tests needs a bunch of refactoring. Anyway, let's fix it consistently, and before that, better to stay consistent with the surrounding code.

Reply:
Fixed in https://github.com/micropython/micropython/commit/f54b3527f20291bc7b158b675bebe39e381067d8